docs/content/documentation/systems/Materials/tensor_mechanics/ComputeLinearElasticPFFractureStress.md

```diff
@@ -2,8 +2,43 @@
 
 # ComputeLinearElasticPFFractureStress
 !syntax description /Materials/ComputeLinearElasticPFFractureStress
+!syntax description /Materials/ComputeLinearElasticPFFractureStress
++This material defines the tensile (positive) and compressive (negative) parts of the strain energy, and their derivatives. It also computes the tensile and compressive parts of the stress. It assumes linear elasticity, but that elasticity tensor can be anisotropic. The main ideas are shown, below:
++1, Calculate the undamaged stress and then decompose the stress into positive and negative parts;
++$$
++\begin{eqnarray}
++  \bf{\sigma} = \mathbf{C} \bf{\epsilon} = \bf{Q} \bf{\Lambda} \bf{Q^{T}} \\
++  = \bf{Q} \bf{\Lambda^{+}}\bf{Q^{T}} + \bf{Q} \bf{\Lambda^{-}}\bf{Q^{T}} \\
++  = \bf{\sigma_{0}^{+}} + \bf{\sigma_{0}^{-}}
++\end{eqnarray}
++$$
++2, Get the positive and negative parts of the strain energy, which are induced by the positive and negative parts of the stress we determined in the first step;
++$$
++\begin{eqnarray}
++  \psi^{+} &=& \frac{1}{2} \bf{\sigma_{0}^{+}} : \bf{\epsilon}\\
++  \psi^{-} &=& \frac{1}{2} \bf{\sigma_{0}^{-}} : \bf{\epsilon}
++\end{eqnarray}
++$$
++3, Get the total free energy for damaged material;
++\begin{eqnarray}
```

Comment:
what's with the extra plusses everywhere? Did you copy paste that out of github?

Reply:
I copied some of the documentation code from the documentation I wrote at the beginning of July when Mike came to INL, maybe that causes this, anyway I will modify this soon.